atomate/vasp/firetasks/exchange_tasks.py

```diff
@@ -0,0 +1,458 @@
+from fireworks import FiretaskBase, FWAction, explicit_serialize
+
+from atomate.utils.utils import get_logger, env_chk
+from atomate.vasp.database import VaspCalcDb
+
+from datetime import datetime
+import numpy as np
+
+from pymatgen import Structure
+from pymatgen.command_line.bader_caller import bader_analysis_from_path
+from pymatgen.analysis.magnetism.heisenberg import HeisenbergMapper, HeisenbergModel
+from pymatgen.analysis.magnetism import CollinearMagneticStructureAnalyzer, Ordering
+from pymatgen.command_line.vampire_caller import VampireCaller
+
+__author__ = "Nathan C. Frey"
+__email__ = "ncfrey@lbl.gov"
+
+logger = get_logger(__name__)
+
+
+@explicit_serialize
+class ExchangeToDB(FiretaskBase):
```

Comment:
Yes haha, I think the original plan was to add some additional info but that wasn't necessary and I never changed it. I'll get rid of it and switch to `MagneticOrderingsToDB`.

Reply:
Ok! In that case, I think it would be useful to have the ability to distinguish magnetic orderings calculated with relaxations vs at a fixed geometry. Could you:

1. Implement the formatting and docstring changes I mentioned above to the `MagneticOrderingsToDB` firetask.
2. Add an option to `MagneticOrderingsToDb` called `additional_fields` which just gets added to the task doc. This is a common pattern in atomate, for example you can copy the implementation of `BoltztrapToDb`.
3. Make your workflow submit additional data indicating that the structure was not relaxed before hand. How you handle I don't have any opinions about.

That way we can easily filter out magnetic structures from the exchange workflow vs magnetism workflow.

@mkhorton Does that sound reasonable to you?

Reply:
I will go ahead and make these changes, it shouldn't be difficult and it will make it easier in the future to modify the static input step.

@utf and @mkhorton

Reply:
I'm on board with this, have been chatting with @ncfrey about our options offline, `MagneticOrderingsToDb` will need to be updated a little but otherwise it should be fine